common/db-scripts/notes/notes-data-load.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+
+if [[ $# -lt 2 ]]; then
+   echo "Usage: $0 <db config> <TENANT> [--clear-loan-audit]";
+   echo "       $0 psql.conf diku";
+
+   exit 1;
+fi
+
+DBCONF=${1}
+export TENANT=${2}
+
+export PGDATABASE=`cat $DBCONF | jq '.database' | cut -d\" -f2`
+export PGUSER=`cat $DBCONF | jq '.username' | cut -d\" -f2`
+export PGPASSWORD=`cat $DBCONF | jq '.password' | cut -d\" -f2`
+export PGHOST=`cat $DBCONF | jq '.host' | cut -d\" -f2`
+export PGPORT=`cat $DBCONF | jq '.port' | cut -d\" -f2`
+
+NOTES="notes.tsv"
```

Comment:
Where is `notes.tsv`? In common/db-scripts/notes/notes.json.tar.gz?

Reply:
Yes. I should make a note of that in the README

Reply:
Noted where to find notes.tsv in this file instead. The README already contains instructions to unzip the file.